examples/arithmetics/src/language-server/arithmetics-util.ts

```diff
@@ -11,7 +11,7 @@ export function applyOp(op: '+' | '-' | '*' | '/' | '^' | '%'): (x: number, y: n
         case '*': return (x, y) => x * y;
         case '^': return (x, y) => Math.pow(x, y);
         case '%': return (x, y) => x % y;
-        default: return (x, y) => {
+        case '/': return (x, y) => {
```

Comment:
Generally speaking, it's always good to have a default case in a switch. Having a concrete case for / is good, but we should add one more default that throws when an unknown operator is passed. If we ever extend the language, that error will tell us where we need to add handling for it.